tests/music_tests.ts

```diff
@@ -2,27 +2,34 @@
  * VexFlow - Music API Tests
  * Copyright Mohit Muthanna 2010 <mohit@muthanna.com>
  */
+import { KeyManager } from '../src/keymanager';
+import { Music } from '../src/music';
 
-VF.Test.Music = (function () {
-  var Music = {
+declare var QUnit: any;
+declare function ok(...args: any): any;
+declare function test(...args: any): any;
+declare function expect(...args: any): any;
+declare function equal(...args: any): any;
+
+export var MusicTests = (function () {
+  return {
     Start: function () {
       QUnit.module('Music');
-      test('Valid Notes', Music.validNotes);
-      test('Valid Keys', Music.validKeys);
-      test('Note Values', Music.noteValue);
-      test('Interval Values', Music.intervalValue);
-      test('Relative Notes', Music.relativeNotes);
-      test('Relative Note Names', Music.relativeNoteNames);
-      test('Canonical Notes', Music.canonicalNotes);
-      test('Canonical Intervals', Music.canonicalNotes);
-      test('Scale Tones', Music.scaleTones);
-      test('Scale Intervals', Music.scaleIntervals);
+      test('Valid Keys', this.validKeys);
+      test('Note Values', this.noteValue);
+      test('Interval Values', this.intervalValue);
+      test('Relative Notes', this.relativeNotes);
+      test('Relative Note Names', this.relativeNoteNames);
+      test('Canonical Notes', this.canonicalNotes);
+      test('Canonical Intervals', this.canonicalNotes);
+      test('Scale Tones', this.scaleTones);
+      test('Scale Intervals', this.scaleIntervals);
     },
 
     validNotes: function () {
       expect(10);
 
-      var music = new VF.Music();
```

Comment:
VF is not a factory, but an Object where Vex.Flow is put. `const VF = Vex.Flow`;

Reply:
I mentioned the factory because there are cases where it is indeed a factory. I refer to the discussion in #857.
I think using `Music` directly is clearer than `VF.Music` or `vf.Music`. Do you agree?

Reply:
I just wanted to specify, that VF.Music is different from vf.Music(). vf.Music() does not need to be replaced because it return new Music() and is perfectly fine with typescript. What is making type checking possible is `import { Music } from '../src/music'`. vf.Music() does not stop you from type checking.